examples/hello_world/src/proto/hello_world_grpc.rs

```diff
@@ -0,0 +1,80 @@
+// This file is generated. Do not edit
+// @generated
+
+// https://github.com/Manishearth/rust-clippy/issues/702
+#![allow(unknown_lints)]
+#![allow(clippy)]
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+
+#![allow(box_pointers)]
+#![allow(dead_code)]
+#![allow(missing_docs)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+#![allow(non_upper_case_globals)]
+#![allow(trivial_casts)]
+#![allow(unsafe_code)]
+#![allow(unused_imports)]
+#![allow(unused_results)]
+
+
+use protobuf::Message;
+use std::io::Write;
+
+// Oak node server interface
+pub trait HelloWorldNode {
+    fn say_hello(&mut self, req: super::hello_world::HelloRequest) -> super::hello_world::HelloResponse;
+
+    fn lots_of_replies(&mut self, req: super::hello_world::HelloRequest) -> Vec<super::hello_world::HelloResponse>;
+
+    fn lots_of_greetings(&mut self, reqs: Vec<super::hello_world::HelloRequest>) -> super::hello_world::HelloResponse;
+
+    fn bidi_hello(&mut self, reqs: Vec<super::hello_world::HelloRequest>) -> Vec<super::hello_world::HelloResponse>;
+}
+
+// Oak node gRPC method dispatcher
+pub fn dispatch(node: &mut HelloWorldNode, grpc_method_name: &str, grpc_channel: &mut oak::Channel) {
+    match grpc_method_name {
+        "/oak.examples.hello_world.HelloWorld/SayHello" => {
```

Comment:
Could this be abstracted in a `MethodInvocation`, parametrized by the input and output types? I guess it may be similar to https://github.com/stepancheg/grpc-rust/blob/master/grpc/src/server/method.rs#L371 .

Reply:
Yeah, probably, but I'm not sure my Rust skills are up to that yet...

Reply:
I've left a TODO in there for now.